README.md

```diff
@@ -92,3 +92,20 @@ The secret being used should be in the JSON format we use for our rotation lambd
 	...
 }
 ```
+### Overriding Default Regions
+
+The default region provider chain is used if no adjustments are made. In order to override this default, use one of the following options listed below.
```

Comment:
These provide a way to override the default region and I do like this change for other use cases (so don't revert it), but we also want a way for customers to configure a backup region in order to provide high availability using replicated secrets. There are multiple ways to accomplish this. Some potential options:

* Allow for a "backupRegion" to be specified via configuration here in the JDBC driver. If present, we can instantiate a second SecretCache with this region. Then in the driver logic, [if we can't retrieve the secret in the primary region](https://github.com/aws/aws-secretsmanager-jdbc/blob/master/src/main/java/com/amazonaws/secretsmanager/sql/AWSSecretsManagerDriver.java#L343) cache, we can fallback to using the backup SecretCache. There are a couple places where the secret is retrieved from the cache so we would want to split out this logic into a helper function.
* We can implement the fallback logic in the SecretCache itself. The cache could take a backup region in via [configuration](https://github.com/aws/aws-secretsmanager-caching-java/blob/master/src/main/java/com/amazonaws/secretsmanager/caching/SecretCacheConfiguration.java) and have two different AWSecretsManager clients. If the primary region fails for whatever reason, we can fallback to the backup region.

In both approaches, if a Secret ARN is provided, we probably want to do some automatic region substitution as well.

Reply:
This comment is spot on with our need. During DR testing, we would like to be able to configure failover secret/region config to be used.